experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/types.ts

```diff
@@ -24,22 +24,17 @@ import {
 	FlexibleNodeContent,
 	Sequence,
 	AssignableFieldKinds,
+	TreeNode,
 } from "../editableTreeTypes";
-import { nodeSym } from "./node";
 
 /**
- * An object-like SharedTree node.  Includes objects, lists, and maps.
+ * An object-like SharedTree node. Includes objects, lists, and maps.
  * @alpha
  */
-export interface SharedTreeNode {
-	// TODO: Make [nodeSym] non-optional when we have factory functions.
-	[nodeSym]?: {
-		on<K extends keyof EditableTreeEvents>(
-			eventName: K,
-			listener: EditableTreeEvents[K],
-		): () => void;
-	};
-}
+export type SharedTreeNode =
```

Comment:
@DLehenbauer, the interface here was no longer useful since I removed `nodeSym`. However, it was still useful to have something that is "one of our node types", so now it's simply a type union of the three.

Reply:
This is better, thanks!